doc/enterprise-shield/p_enterprise_shield_cluster.md

```diff
@@ -0,0 +1,132 @@
+
+In Figure 1, there are two Gateways running in the DMZ and a matching set of two Gateways on the trusted network. Thus, each internal Gateway is paired to one explicit cluster member in the DMZ.
+
+![Gateway Topology Showing Reverse Connectivity in a Cluster](../images/f-es-maxsecurity-cluster.png)
+
+**Figure 1: Enterprise Shield Topology Configured for High Availability**
+
+The key to making the Enterprise Shield configuration highly available is to configure clustering on each cluster member, and have an equivalent number of cluster members in the DMZ and the trusted network.
+
+The following steps provide a high-level overview about cluster configuration. See [Configure a KAAZING Gateway Cluster](../high-availability/p_high_availability_cluster.md) for detailed information about cluster configuration and the [cluster](../admin-reference/r_configure_gateway_cluster.md) element.
+
+-   Update the properties element to use a numbered value for the DMZ backplane variable---such as `gateway1`---to provide a unique name for the Gateway cluster pair:
+
+    **Internal Gateway**
+
+    ``` auto-links:
+    <properties>
+
+      <!-- The Gateway in the DMZ to connect to. -->
+      <property>
+        <name>dmz.cluster.backplane.hostname</name>
+        <value>gateway1.example.dmz.net</value>
+      </property>
+
+    </properties>
+    ```
+
+    **DMZ Gateway**
+
+    ``` auto-links:
+    <properties>
+      <!-- The IP address of this cluster member. -->
+      <property>
+        <name>gateway.ip.address</name>
+        <value>172.19.20.1</value>
+      </property>
+
+      <!-- The port number (value) must be different for each member. -->
+      <property>
+        <name>cluster.port</name>
+        <value>5941</value>
+      </property>
+
+      <!-- The publicly addressable hostname. -->
+      <property>
+        <name>dmz.cluster.frontplane.hostname</name>
+        <value>gateway1.example.com</value>
+      </property>
+
+      <!-- The private address that the internal gateway connects to. -->
+      <property>
+        <name>dmz.cluster.backplane.hostname</name>
+         <value>gateway1.example.dmz.net</value>
+      </property>
+
+    </properties>
+    ```
+
+-   On the DMZ Gateway, add the cluster element:
+
+    ``` auto-links:
+    <cluster>
+      <name>tcp://${gateway.ip.address}:${cluster.port}</name>
+      <accept>172.19.20.1</accept>
+      <connect>udp://224.2.2.44:44444</connect>
+    </cluster>
+    ```
+
+    The accept element contains the URI on which the cluster member listens for other cluster members. The connect element contains the URI used to discover other cluster members. In this case, it uses a multicast address.
+
+-   On the DMZ Gateway, add a balancer service for each service. The following example shows the balancer service for DMZ App1:
+
+    ``` auto-links:
+    <service>
+
+      <name>Balancer for DMZ App1</name>
+      <accept>wss://gateway.example.com:8443/app1</accept>
```

Comment:
I just notice the port number. We should change all references of 8443 to 443 in all doc files.